blobnfs-troubleshoot.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Need super user access to certain commands
+if [ $(id -u) -ne 0 ]; then
+    echo "Run this script as root!"
+    exit 1
+fi
+
+# Default branch name
+branchname="develop"
+
+# Check for the number of parameters to the script
+if [[ $# -lt 2 ]]; then
+    read -p "Enter the absolute mount path: " absmountpoint
+    read -p "Enter the server IP: " serverip
```

Comment:
don't ask serverip, get it from mountmap in case of aznfs and o/w in case of nfs mount

Reply:
also we should check for non-aznfs mounts and bail out with appropriate error and doc references.